src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -60,6 +61,7 @@ function AxisSystem(props: Props): JSX.Element {
         gridTemplateRows: `${axisOffsets.top}px 1fr ${axisOffsets.bottom}px`,
       }}
     >
+      {title !== undefined && <div className={styles.title}>{title}</div>}
```

Comment:
The `undefined` check may not be required (just `!title`), since it doesn't really make sense to show an empty string.

Reply:
Rather `title && ...`, you mean ?

Reply:
euh yes